.gitignore

```diff
@@ -0,0 +1,5 @@
+# build folder
+build/*
+
+# MacOS
+.DS_Store
```

Comment:
Maybe starting from the the default C++ gitignore? https://github.com/github/gitignore/blob/master/C%2B%2B.gitignore

Reply:
I would keep it basic for the time being.